chain/tendermint/events.go

```diff
@@ -1,24 +1,22 @@
 package tendermint
 
 import (
-	"bytes"
-
 	abcitypes "github.com/tendermint/tendermint/abci/types"
 )
 
 // AttributeValue returns an event attribute value given the eventType and attribute key tuple.
 // In the event of duplicate types and keys, returns the first attribute value found.
-// If not found, ok is false.
-func AttributeValue(events []abcitypes.Event, eventType string, attrKey []byte) (found []byte, ok bool) {
+// If not found, returns empty string and false.
+func AttributeValue(events []abcitypes.Event, eventType, attrKey string) (string, bool) {
```

Comment:
Even though the attribute data structure is `{Key: []byte, Value: []byte}`. The consumers I've noticed in cosmos want them as strings.